roles/tftp-server/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: install a web server to host late_command.sh
+  apt:
+    name: "{{ item }}"
+  with_items:
+  - nginx
+  - python
```

Comment:
wat. Surely we had python, if we were running ansible? Also, what uses that, here?